packages/backend/src/controllers/slackController.ts

```diff
@@ -60,11 +61,11 @@ export class SlackController extends BaseController {
         unauthorisedInDemo,
     ])
     @SuccessResponse('200', 'Success')
-    @Put('/notification-channel')
+    @Put('/custom-settings')
     @OperationId('UpdateNotificationChannel')
     async updateNotificationChannel(
         @Request() req: express.Request,
-        @Body() body: { channelId: string | null },
+        @Body() body: SlackAppCustomSettings,
```

Comment:
we need to run `generate-api`

Reply:
nvm, its already done. was a comment to self

Reply:
I just ran it and there were no changes, should be good 👌